zeppelin-server/src/main/java/org/apache/zeppelin/rest/LoginRestApi.java

```diff
@@ -126,8 +123,9 @@ public Response logout() {
     data.put("principal", "anonymous");
     data.put("roles", "");
     data.put("ticket", "anonymous");
+    data.put("WWW-Authenticate", "Basic realm=\"Login required\"");
```

Comment:
Having `WWW-Authenticate` in the body of the response, will not solve the logout issue, it will be ignored by the browser, you have to put this information in the [response header](https://tools.ietf.org/html/rfc2617#section-3.2.1).

Plus after reading this method, I think something is not right here. In my understanding if you want to logout a Subject you should use the getSecurityManager.

somthing like
```
Subject subject = SecurityUtils.getSubject();
SecurityUtils.getSecurityManager().logout(subject);
```